frontend/src/Settings/Indexers/Indexers/Indexers.js

```diff
@@ -76,6 +76,7 @@ class Indexers extends Component {
                   <Indexer
                     key={item.id}
                     {...item}
+                    showPriority={items.some((index) => index.priority != 25)}
```

Comment:
Compute and store this outside of the map so it isn't recalculated for each item.

```
const showPriority = items.some((index) => index.priority != 25)
...
showPriority={showPriority}
```